src/libs/actions/Report.ts

```diff
@@ -2308,6 +2308,8 @@ function inviteToRoom(reportID: string, inviteeEmailsToAccountIDs: Record<string
 
     const logins = inviteeEmails.map((memberLogin) => OptionsListUtils.addSMSDomainIfPhoneNumber(memberLogin));
     const newPersonalDetailsOnyxData = PersonalDetailsUtils.getNewPersonalDetailsOnyxData(logins, inviteeAccountIDs);
+    const pendingAccountIDs = inviteeAccountIDs.map((accountID) => ({accountID: accountID.toString(), pendingAction: CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD}));
+    const pendingVisibleChatMembers = [...(report?.pendingVisibleChatMembers ?? []), ...pendingAccountIDs];
```

Comment:
this is kind of repetitive i wonder if we should create a function for this.

Reply:
Thanks, I created a new utils function to dry code